src/marlin_stubs/pause/pause.cpp

```diff
@@ -276,7 +302,7 @@ bool unload_filament(const float &unload_length, const bool show_lcd /*=false*/,
     constexpr size_t ramUnloadSeqSize = sizeof(ramUnloadSeq) / sizeof(RamUnloadSeqItem);
 
     //cannot draw progress in plan_pause_e_move, so just change phase to ramming
-    fsm_change(ClinetFSM::Load_unload, GetPhaseIndex(PhasesLoadUnload::Ramming), 50, 0);
+    fsm_change(ClinetFSM::Load_unload, PhasesLoadUnload::Ramming, 50, 0);
```

Comment:
Why isn't fsm an object (class) encapsulating all its operations?

Reply:
because marlin client is written in C